website/client/assets/scss/item.scss

```diff
@@ -64,4 +64,26 @@
   text-align: center;
   color: $gray-400;
   margin-top: 4px;
-}
\ No newline at end of file
+}
+
+.item > .badge {
```

Comment:
I think this code got added by the merge, but it's not used anymore in the current `develop` https://github.com/HabitRPG/habitica/blob/develop/website/client/assets/scss/item.scss